client/src/components/WorkflowInvocationState/WorkflowInvocationBiocompute.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div v-if="Object.keys(invocation).length">
+        <details class="invocation-biocompute-objects">
+            <summary><b>Biocompute Object</b></summary>
+            <div v-if="Object.keys(invocation).length">
+                <details class="invocation-biocompute-object-export">
+                    <summary><b>Export BioCompute Object</b></summary>
+                    <div>
+                        <form v-on:submit.prevent="submitForm">
+                            <div class="form-group">
+                                <label for="fetch">BCO DB Root URL</label>
+                                <input
+                                    type="text"
+                                    class="form-control"
+                                    id="fetch"
+                                    placeholder="https://beta.portal.aws.biochemistry.gwu.edu"
+                                    v-model="form.fetch"
+                                />
+                            </div>
+                            <div class="form-group">
+                                <label for="authorization">User API Key</label>
+                                <input
+                                    type="text"
+                                    class="form-control"
+                                    id="authorization"
+                                    v-model="form.authorization"
+                                />
+                            </div>
+                            <div class="form-group">
+                                <label for="table">Table</label>
+                                <input
+                                    type="text"
+                                    class="form-control"
+                                    id="table"
+                                    placeholder="bco_draft"
+                                    v-model="form.table"
+                                />
+                            </div>
+                            <div class="form-group">
+                                <label for="owner_group">Group</label>
+                                <input
+                                    type="text"
+                                    class="form-control"
+                                    id="owner_group"
+                                    placeholder="bco_drafters"
+                                    v-model="form.owner_group"
+                                />
+                            </div>
+                            <div class="form-group">
+                                <button class="btn btn-primary">Submit</button>
```

Comment:
Assuming these are mounted with the standard component mount, you should have access to localization features. It's probably a  good habit for all of us to start doing by default:
```html
Option 1: as a directive
<button class="btn btn-primary" v-localize>Submit</button>

Option 2: as a filter:
<button class="btn btn-primary" v-localize>{{ 'Submit' | localize }}</button>
````